test/wallaby/browser/dialog_test.exs

```diff
@@ -0,0 +1,208 @@
+defmodule Wallaby.Browser.DialogTest do
+  use Wallaby.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "dialogs.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "accept_dialogs/1" do
+    test "accept window.alert", %{page: page} do
+      result = page
+      |> accept_dialogs()
+      |> click(Query.link("Alert"))
+      |> find(Query.css("#result"))
+      |> Element.text()
```

Comment:
I went with the the second version now, let me know if you had something else in mind @keathley.